opencraft/settings.py

```diff
@@ -0,0 +1,340 @@
+
+DEBUG = env.bool('DEBUG', default=False)
+
+LOGIN_URL = '/admin/login/'
+
+# Database ####################################################################
+
+# Set via the environment variable `DATABASE_URL`
+DATABASES = {
+    'default': env.db(),
+}
+
+
+# Application definition ######################################################
+
+LOCAL_APPS = (
+    'api',
+    'instance',
+)
+
+INSTALLED_APPS = (
+    'grappelli',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'django_extensions',
+    'foundation',
+    'compressor',
+    'djangular',
+    'rest_framework',
+    'huey.djhuey',
+    'swampdragon',
+    'debug_toolbar',
+) + LOCAL_APPS
+
+MIDDLEWARE_CLASSES = (
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'django.middleware.security.SecurityMiddleware',
+)
+
+ROOT_URLCONF = 'opencraft.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [root(p, 'templates') for p in [''] + list(LOCAL_APPS)],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+                'django.core.context_processors.request',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'opencraft.wsgi.application'
+
+
+# Internationalization ########################################################
+
+# https://docs.djangoproject.com/en/1.8/topics/i18n/
+
+LANGUAGE_CODE = 'en-us'
+TIME_ZONE = 'UTC'
+USE_I18N = True
+USE_L10N = True
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images) ######################################
+
+# https://docs.djangoproject.com/en/1.8/howto/static-files/
+
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    # other finders..
+    'compressor.finders.CompressorFinder',
+)
+
+STATICFILES_DIRS = (
+    root('static'),
+)
+
+STATIC_ROOT = root('build/static')
+STATIC_URL = '/static/'
+
+COMPRESS_PRECOMPILERS = (
+    ('text/x-scss', 'django_libsass.SassCompiler'),
+)
+
+
+# Django-extensions ###########################################################
+
+SHELL_PLUS = "ipython"
+RUNSERVERPLUS_SERVER_ADDRESS_PORT = '127.0.0.1:5000'
```

Comment:
@antoviaque Can we wrap this setting in `env(...)`, too? I'm trying to run the server in a vagrant box without port forwarding so accessing it through `127.0.0.1` doesn't work.